core/templates/pages/topic-viewer-page/topic-story-section/adventure-end-test-card.component.css

```
@@ -0,0 +1,266 @@
+}
+
+.adventure-end-test-card-description {
+  color: #3d3d3d;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 24px;
+  margin: 0;
+  max-width: 520px;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)